core/tests/tests/Test.mint

```diff
@@ -17,3 +17,25 @@ suite "Test (Function)" {
     test() == ""
   }
 }
+
+suite "Test with HTML reference" {
+  test "it works" {
+    <button as button/>
+    |> Test.Html.start
+    |> Test.Context.map((item : Dom.Element) { button != Maybe.Nothing })
```

Comment:
Because we need to render the button on the page (and remove it when the test finishes), [`Test.Html.start`](https://github.com/mint-lang/mint/blob/master/core/source/Test/Html.mint#L10-L22) does that using a `Test.Context`.

Maybe in the future, we can figure out something that is less convoluted and easier to understand.